remediation/code-samples/01-countEvens.js

```diff
@@ -39,7 +42,7 @@ function check(output, expected)
 if (require.main === module) {
   console.log('Running sanity checks for countEvens:');
   console.log(check(countEvens([1,2,3,4]),2));
-  console.log(check(countEvens([2,2,8,4]),2));
+  console.log(check(countEvens([2,2,8,4]),2)); // PLEASE REVIEW
```

Comment:
The second argument our `check()` function takes is the expected number of even elements. In the array` [2, 2, 4, 8] `there should be four expected even elements.